limit/id.go

```diff
@@ -7,9 +7,11 @@ type ID string
 
 // IDs of configurable limits.
 const (
-	NotificationRulesPerUser    ID = "notification_rules_per_user"
-	ContactMethodsPerUser       ID = "contact_methods_per_user"
-	EPStepsPerPolicy            ID = "ep_steps_per_policy"
+	NotificationRulesPerUser ID = "notification_rules_per_user"
+	ContactMethodsPerUser    ID = "contact_methods_per_user"
+	// Maximum steps on a single escalation policy step
+	EPStepsPerPolicy ID = "ep_steps_per_policy"
+	// Maximum actions on a single escalation policy step
 	EPActionsPerStep            ID = "ep_actions_per_step"
 	ParticipantsPerRotation     ID = "participants_per_rotation"
 	RulesPerSchedule            ID = "rules_per_schedule"
```

Comment:
We should add descriptions for some of these others:
- `RulesPerSchedule` pertains to all rules for all assignments/targets or per assignment/target, can't remember -- but we should add a description indicating exactly how it works
- `UnackedAlertsPerService`, only affects newly created alerts, but not re-escalated ones
- `UserOverridesPerSchedule` only limits future overrides (e.g. end in the future)

We could probably describe each for consistency, but those 3 come to mind as being not totally clear